test/tests/interfacekernels/InterfaceSorption/tests

```diff
@@ -121,10 +116,243 @@
                 Materials/properties_2/type=ADGenericConstantMaterial
                 Postprocessors/flux_inner/type=ADSideDiffusiveFluxIntegral
                 Postprocessors/flux_outer/type=ADSideDiffusiveFluxIntegral
-                Outputs/file_base=interface_sorption_Henry_out'
-    exodiff = 'interface_sorption_Henry_out.e'
-    prereq = Henry_ad
+                Outputs/file_base=interface_sorption_Henry_ad_penalty_out'
+    exodiff = 'interface_sorption_Henry_ad_penalty_out.e'
     abs_zero = 1e-7
     mesh_mode = REPLICATED
   []
+
+  [Sievert_transient_non_ad]
+    requirement = 'The system shall have the capability to enforce interfacial conditions based on the Sievert law in isothermal conditions during transient.'
+    type = 'Exodiff'
+    input = 'interface_sorption_transient.i'
+    cli_args = 'Outputs/file_base=interface_sorption_transient_Sievert_non_ad_out'
+    exodiff = 'interface_sorption_transient_Sievert_non_ad_out.e'
+    mesh_mode = REPLICATED
+    recover = false
```

Comment:
Make a note here referring to the issue number you made for this setting.

Reply:
Done